### PR TITLE
ISDB-287 - Upload document screen changes

### PR DESCRIPTION
**Pull request by parvezhmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ISDB-287

### Change description ###

- user will get error when continue without uploading a file
- user will get error after clicking of upload button, in case no file is selected.
- remove document management url.
- To call fis cos api, only one base url will be used.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by hmcts-kodatala — APPROVED** [review]
TOTAL_FILES_EXCEED_ERROR:file count should be more than 5 , pls check this count and file count should replace from config in error msg.

File: src/main/resources/upload-your-documents/translation.json [hunk 1]
```diff
@@ -25,12 +25,16 @@
     "en": {
     "SIZE_ERROR": "File size exceeds 20Mb. Please upload a file that is less than 20Mb",
     "FORMAT_ERROR": "This service only accepts files in the formats - Ms Word, Ms Excel, PDF, JPG, GIF, PNG, TXT, RTF",
-    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry"
+    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry",
```
Comment (yogendraU) [inline src/main/resources/upload-your-documents/translation.json:28]: is this only 5 Files max?

File: src/main/resources/upload-your-documents/translation.json [hunk 2]
```diff
@@ -25,12 +25,16 @@
...
+    "CONTINUE_WITHOUT_UPLOAD_ERROR": "You cannot continue without uploading the application",
+    "NO_FILE_UPLOAD_ERROR": "Please choose a file to upload"
     },
     "cy": {
     "SIZE_ERROR": "File size exceeds 20Mb. Please upload a file that is less than 20Mb - welsh",
     "FORMAT_ERROR": "This service only accepts files in the formats - Ms Word, Ms Excel, PDF, JPG, GIF, PNG, TXT, RTF - welsh",
-    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry - welsh"
+    "TOTAL_FILES_EXCEED_ERROR": "You can upload 5 files only. Please delete one of the uploaded files and retry - welsh",
```
Comment (yogendraU) [inline src/main/resources/upload-your-documents/translation.json:35]: Please check with Subha and update the story with number of files allowable. I guess the Defect ISDB-280 is already created for that.

**Review by yogendraU — CHANGES_REQUESTED** [review]

**Review by yogendraU — APPROVED** [review]

File: src/main/resources/upload-your-documents/translation.json (hunk 2 above)
Comment (parvezhmcts) [reply src/main/resources/upload-your-documents/translation.json:35]: Already discussed.

**Review by Murshadhaque — APPROVED** [review]

**Pull request merged**